src/components/Slide/player.css

```
@@ -423,10 +375,7 @@
 
 .slide--mobile-device .slide-interaction-overlay {
   --slide-player-bottom-offset: 0px;
-  transform: translate(
-    var(--slide-interaction-drag-x, 0px),
-    var(--slide-interaction-drag-y, 0px)
-  );
+  transform: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)